books/management/commands/stud_commands.py

```diff
@@ -0,0 +1,85 @@
+from django.core.management import BaseCommand
+from students.models import Students, Lessons, Teachers, LessonStudent
+from django.db.models import Count
+from django.db.models import Avg, Max
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        # Усі студенти чає прізвище починається з літери "S"
+        students = Students.objects.filter(last_name__startswith='S')
+        for student in students:
+            print(student.first_name, student.last_name)
+
+        # Усі студенти які отримали оцінку 8 або більше за певну дату.
+
+        lesson_students = LessonStudent.objects.filter(mark__gte=8, lesson__date_time__startswith='2022-11-21')
```

Comment:
краще використовувати __date:

```suggestion
        lesson_students = LessonStudent.objects.filter(mark__gte=8, lesson__date_time__startswith='2022-11-21')
```